packages/FormElement/src/components/Help/Help.js

```diff
@@ -5,11 +5,11 @@ import useI18n from "@paprika/l10n/lib/useI18n";
 import * as sc from "./Help.styles";
 
 function Help(props) {
-  const { children, isDisabled, a11yText } = props;
+  const { children, isDisabled, a11yText, align } = props;
```

Comment:
I would suggest just adding `...moreProps` here instead of `align`, and then spreading that onto the `<sc.Help>` (which is actually a styled `<Popover>`).  Then we can propagate any other Popover props we might want to in the future.

You can even leave `align={align: Popover.types.align.TOP}` on `<sc.Help>` _before_ `...moreProps` as the "default" value, but it will get overridden if the user passes their own `align`.

Reply:
Oh... nevermind, I see now that `<Help>` isn't exposed to the end user, it's just wrapping the `<Label>` `help` prop.... 🤔